### PR TITLE
Fix hexbin and remove data/ directory

### LINKED ISSUE #250: Remove data directory and move all data to _data directory

### PR DESCRIPTION
**Pull request by wd15**
Fix #250, #245, #227, #190, #249, #238

### CODE REVIEW

File: _data/hexbin.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 def hexbin_yaml_to_json():
     """Generate JSON image data from the YAML.
     """
-    data = yaml.load(open('data/hexbin.yaml', 'r'))
+    data = yaml.load(open('_data/hexbin.yaml', 'r'))
```
Comment (tkphd) [inline _data/hexbin.py]: My build fails, line 35 in this file refers to ```data/hexbin.json``` erroneously.

File: _includes/head.html [hunk 2]
```diff
@@ -19,8 +19,10 @@
 
 <!-- D3 and vega -->
 <script type="text/javascript" src="{{ site.baseurl }}/js/d3.v4.min.js"></script>
+<!-- <script src="//d3js.org/d3.v3.min.js" charset="utf-8"></script> -->
```
Comment (tkphd) [inline _includes/head.html]: Cruft or breadcrumbs?

File: benchmarks/benchmark1.ipynb [hunk 3]
```diff
@@ -1187,7 +1187,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The results required to compare phase field codes are still being evalutated. However, the initial  comparison scheme will include the global free energy as a function of time and the $c$ field at various snapshot times ($t = 1, 5, 10, 20, 100, 200, 500, 1000, 2000, 3000, 10000$). Results of studies between different codes will be linked to from here."
+    "Go to the [simulation result page]({{ site.baseurl }}/simulations) to simulation results from different codes."
```
Comment (tkphd) [inline benchmarks/benchmark1.ipynb]: Check your grammar here.

File: benchmarks/benchmark1.ipynb.raw.html [hunk 4]
```diff
@@ -4040,7 +4040,7 @@ <h2 id="Results">Results<a class="anchor-link" href="#Results">&#182;</a></h2>
 </div>
 <div class="inner_cell">
 <div class="text_cell_render border-box-sizing rendered_html">
-<p>The results required to compare phase field codes are still being evalutated. However, the initial  comparison scheme will include the global free energy as a function of time and the $c$ field at various snapshot times ($t = 1, 5, 10, 20, 100, 200, 500, 1000, 2000, 3000, 10000$). Results of studies between different codes will be linked to from here.</p>
+<p>Go to the <a href="{{ site.baseurl }}/simulations">simulation result page</a> to simulation results from different codes.</p>
```
Comment (tkphd) [inline benchmarks/benchmark1.ipynb.raw.html]: Check your grammar here.

File: benchmarks/benchmark2.ipynb [hunk 5]
```diff
@@ -1592,7 +1592,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The results required to compare phase field codes are still being evalutated. However, the initial  comparison scheme will include the global free energy as a function of time and the $c$ field at various snapshot times ($t = 1, 5, 10, 20, 100, 200, 500, 1000, 2000, 3000, 10000$). Results of studies between different codes will be linked to from here."
+    "Go to the [simulation result page]({{ site.baseurl }}/simulations) to simulation results from different codes."
```
Comment (tkphd) [inline benchmarks/benchmark2.ipynb]: Check your grammar here.

File: benchmarks/benchmark2.ipynb.raw.html [hunk 6]
```diff
@@ -8701,7 +8701,7 @@ <h2 id="Results">Results<a class="anchor-link" href="#Results">&#182;</a></h2>
 </div>
 <div class="inner_cell">
 <div class="text_cell_render border-box-sizing rendered_html">
-<p>The results required to compare phase field codes are still being evalutated. However, the initial  comparison scheme will include the global free energy as a function of time and the $c$ field at various snapshot times ($t = 1, 5, 10, 20, 100, 200, 500, 1000, 2000, 3000, 10000$). Results of studies between different codes will be linked to from here.</p>
+<p>Go to the <a href="{{ site.baseurl }}/simulations">simulation result page</a> to simulation results from different codes.</p>
```
Comment (tkphd) [inline benchmarks/benchmark2.ipynb.raw.html]: Check your grammar here.

**Comment by wd15** [comment]
@tkphd should be fixed now, if so please merge.

**Pull request merged**